drivers/phy/cadence/phy-cadence-torrent.c

```diff
@@ -1370,9 +1647,15 @@ static int cdns_torrent_dp_configure(struct phy *phy, union phy_configure_opts *
 	return ret;
 }
 
+static const struct cdns_torrent_data cdns_map_torrent = {
+	0x0,
+	0x1,
+};
+
```

Comment:
Will the reads and writes work fine on Cadence Simulation environment? Are the register offsets same between TI and Cadence environment?